packages/kbn-logging-mocks/BUILD.bazel

```diff
@@ -0,0 +1,106 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
```

Comment:
Should the Core team own this package as we do with `kbn-logging`? 🤔

Reply:
I believe it makes sense. I've added the team to the code owners. 👍